symengine/lib/symengine_wrapper.pyx

```diff
@@ -38,6 +39,15 @@ include "config.pxi"
 class SympifyError(Exception):
     pass
 
+cpdef object capsule_to_basic(object capsule):
+    cdef CRCPBasic *p = <CRCPBasic*>PyCapsule_GetPointer(capsule, NULL)
+    return c2py(p.m)
+
+cpdef void basic_to_capsule(object capsule, object basic):
```

Comment:
Can the method signature be something like,
```suggestion
cpdef object basic_to_capsule(Basic basic):
```
?

i.e. can `reticulate` get a pointer from the pycapsule created by symengine.py?

Reply:
Nvm, you want the memory to be managed by R

Reply:
`assign_to_capsule` was a better name now that I think of it.
```suggestion
cpdef void assign_to_capsule(object capsule, object value):
```

Reply:
I think it could, but the owner and lifetime of this pointer becomes unclear for me. Currently this pointer inside capsule is created from R (with `basic_new_heap`) and will be deleted from R, otherwise we need to let python to hold reference to this object and it would be more complicated.

Reply:
Good to know.